app/main/controllers/template/rtmedia-functions.php

```diff
@@ -1467,9 +1467,12 @@ function rtmedia_pagination_next_link() {
  */
 function rtmedia_pagination_page_link( $page_no = '' ) {
 
-	global $rtmedia_interaction, $rtmedia_query, $post, $rtmedia;
+	global $rtmedia_interaction, $rtmedia_query, $post, $rtmedia, $wp_query;
 
 	$wp_default_context = array( 'page', 'post' );
+	$rtm_context        = $wp_query->query_vars['rtm_context'];
+	$rtm_attr           = $wp_query->query_vars['rtm_attr'];
+	$rtm_term           = $wp_query->query_vars['rtm_term'];
```

Comment:
@Sidsector9 Please use `get_query_var()` to get values of query_vars.

https://codex.wordpress.org/Function_Reference/get_query_var